examples/all-clusters-app/nrfconnect/README.md

```diff
@@ -178,100 +178,58 @@ using the following command:
 
         $ git submodule update --init
 
-The example requires a specific revision of the nRF Connect SDK. You can either
-install it along with the related tools directly on your system or use a Docker
-image that has the tools pre-installed.
-
-If you are a macOS user, you won't be able to use the Docker container to flash
-the application onto a Nordic development kit due to
-[certain limitations of Docker for macOS](https://docs.docker.com/docker-for-mac/faqs/#can-i-pass-through-a-usb-device-to-a-container).
-Use the [native shell](#using-native-shell-for-setup) for building instead.
-
-### Using Docker container for setup
-
-To use the Docker container for setup, complete the following steps:
-
-1.  If you do not have the nRF Connect SDK installed yet, create a directory for
-    it by running the following command:
-
-        $ mkdir ~/nrfconnect
+> **Note**:
+>
+> For Linux operating system
+> Install [SEGGER J-Link Software](https://www.segger.com/downloads/jlink/#J-LinkSoftwareAndDocumentationPack).
 
-2.  Download the latest version of the nRF Connect SDK Docker image by running
-    the following command:
+### Installing command line requirements
 
-        $ docker pull nordicsemi/nrfconnect-chip
+To build on the command line, complete the following steps:
```

Comment:
```suggestion
To be able to build the application on the command line, complete the following steps:
```

Reply:
Just realized that we can delete this whole line bc step is only one. Thanks